### DISCUSSION TITLE
2.0.8 какой режим включен, auto, list или all

### DISCUSSION THREAD

**Discussion by red-ut-dm** [Anonym-tsk/nfqws-keenetic#42] [Обсуждения]
подскажите, как изменить режим работы в 2.0.8? в 2.0.7 можно было выбирать режим работы во время установки. как изменить режим в 2.0.8

**Comment by Nik-Targon** [comment]
В версии 2.0.8 режим работы по умолчанию auto. Изменить можно редактируя файл конфига /opt/etc/nfqws/nfqws.conf

**Reply by Neytrino-OnLine** [reply]
У меня сохранилась старая инструкция, и получилось вот так:
cd nfqws-keenetic
./install.sh

p.s. если кому-то надо удалить старую версию (установленную вручную):
./nfqws-keenetic/uninstall.sh
